src/config/localization/translations.json

```diff
@@ -376,7 +376,7 @@
   "Locked Price": "Locked Price",
   "Last Price": "Last Price",
   "Closed Price": "Closed Price",
-  "Win": "Win",
```

Comment:
Again, only add new translations here, do not delete existing one unless you checked there is no reference to it.  Otherwise test / build may fail.